src/transformers/trainer.py

```diff
@@ -758,17 +759,20 @@ def _wrap_model(self, model, training=True):
 
     def train(
         self,
-        resume_from_checkpoint: Optional[str] = None,
+        resume_from_checkpoint: Optional[Union[str, bool]] = None,
         trial: Union["optuna.Trial", Dict[str, Any]] = None,
         **kwargs,
     ):
         """
         Main training entry point.
 
         Args:
-            resume_from_checkpoint (:obj:`str`, `optional`):
-                Local path to a saved checkpoint as saved by a previous instance of :class:`~transformers.Trainer`. If
-                present, training will resume from the model/optimizer/scheduler states loaded here.
+            resume_from_checkpoint (:obj:`str` or :obj:`bool`, `optional`):
+
+                    If a :obj:`str`, local path to a saved checkpoint as saved by a previous instance of
+                    :class:`~transformers.Trainer`. If a :obj:`bool` and equals `True`, load the last checkpoint in
+                    `args.output_dir` as saved by a previous instance of :class:`~transformers.Trainer`. If present,
+                    training will resume from the model/optimizer/scheduler states loaded here.
```

Comment:
```suggestion
                If a :obj:`str`, local path to a saved checkpoint as saved by a previous instance of
                :class:`~transformers.Trainer`. If a :obj:`bool` and equals `True`, load the last checkpoint in
                `args.output_dir` as saved by a previous instance of :class:`~transformers.Trainer`. If present,
                training will resume from the model/optimizer/scheduler states loaded here.
```
No need for one more level of indentation or the new line now :-)

Reply:
Oops. Thanks for pointing! :-)